packer/flow.go

```diff
@@ -89,6 +93,17 @@ func (f *Flow) hasTx(txid thor.Bytes32, txBlockRef uint32) (bool, error) {
 	return f.runtime.Chain().HasTransaction(txid, txBlockRef)
 }
 
+func (f *Flow) maxPriorityFeePerGasTooLow(t *tx.Transaction) error {
```

Comment:
```suggestion
func (f *Flow) validatePriorityFeePerGas(t *tx.Transaction) error {
```

Reply:
I think the current name makes more sense since it is more specific. `validatePriorityFeePerGas` does not express that we are validating a `maxPriorityFeePerGas` value is >= than the set in the flag.

Reply:
For function names, I think verbs is preferred instead of nouns, if you prefer `maxPriorityFeePerGasTooLow` then it should be `isMaxPriorityFeePerGasTooLow(t *tx.Transaction) bool`.

Reply:
Ok, I can use `is...`. Won't change it just now until the staff related to the effective priority fee gets clarified, because it will affect the name of this method.